recipes/default.rb

```diff
@@ -106,9 +117,26 @@
   )
 end
 
+# Create zone files if the server is a master server
+if node['bind']['zonetype'] == 'master'
+  all_zones.sort.uniq.each do |zone|
+    template "#{node['bind']['vardir']}/master/db.#{zone}" do
```

Comment:
A LWRP or HWRP would be very nice. As a first step I would leave this changes so there is a elementary way to add zones to the server. In a next step we could try to implement a LWRP for zone management.